blocks/global-phrases-block/intl.json

```diff
@@ -1,48 +1,62 @@
 {
-   "global.gallery-expand-button":{
-      "en":"Expand",
-      "sv":"Förstora",
-      "no":"Utvid",
-      "fr":"Développer",
-      "de": "Erweitern",
-      "es": "Expandir",
-      "ja": "展開",
-      "ko": "더보기"
-   },
-   "global.gallery-page-count-text":{
-      "en":"<span class='sr-only'>Image</span> %{current} of %{total}",
-      "sv":"%{current} av %{total}",
-      "no":"%{current} av %{total}",
-      "fr":"%{current} de %{total}",
-      "de": "%{current} von %{total}",
-      "es": "%{current} de %{total}",
-      "ja": "%{current} / %{total}",
-      "ko": "%{current}/%개 보는 %{total}"
-   },
-   "global.gallery-autoplay-button":{
-      "en":"Autoplay",
-      "sv":"Spela upp",
-      "no":"Spill av",
-      "fr":"Lecture automatique",
-      "de": "Automatische Wiedergabe",
-      "es": "Reproducción automática",
-      "ja": "自動再生",
-      "ko": "자동 재생"
-   },
-   "global.gallery-pause-autoplay-button":{
-      "en":"Pause autoplay",
-      "sv":"Pausa",
-      "no":"Stopp",
-      "fr":"Mettre en pause la lecture automatique",
-      "de": "Automatische Wiedergabe anhalten",
-      "es": "Pausar reproducción automática",
-      "ja": "自動再生の一時停止",
-      "ko": "자동 재생 일시 중지"
-   },
-   "global.gallery-autoplay-label-start": {
-     "en": "Start automatic slide show"
-   },
-   "global.gallery-autoplay-label-stop": {
-     "en": "Stop automatic slide show"
-   }
-}
+	"global.gallery-expand-button": {
+		"de": "Erweitern",
+		"en": "Expand",
+		"es": "Expandir",
+		"fr": "Développer",
+		"ja": "展開",
+		"ko": "더보기",
+		"no": "Utvid",
+		"sv": "Förstora"
+	},
+	"global.gallery-page-count-text": {
+		"de": "%{current} von %{total}",
+		"en": "<span class='sr-only'>Image</span> %{current} of %{total}",
+		"es": "%{current} de %{total}",
+		"fr": "%{current} de %{total}",
+		"ja": "%{current} / %{total}",
+		"ko": "%{current}/%개 보는 %{total}",
+		"no": "%{current} av %{total}",
+		"sv": "%{current} av %{total}"
+	},
+	"global.gallery-autoplay-button": {
+		"de": "Automatische Wiedergabe",
+		"en": "Autoplay",
+		"es": "Reproducción automática",
+		"fr": "Lecture automatique",
+		"ja": "自動再生",
+		"ko": "자동 재생",
+		"no": "Spill av",
+		"sv": "Spela upp"
+	},
+	"global.gallery-pause-autoplay-button": {
+		"de": "Automatische Wiedergabe anhalten",
+		"en": "Pause autoplay",
+		"es": "Pausar reproducción automática",
+		"fr": "Mettre en pause la lecture automatique",
+		"ja": "自動再生の一時停止",
+		"ko": "자동 재생 일시 중지",
+		"no": "Stopp",
+		"sv": "Pausa"
+	},
+	"global.gallery-autoplay-label-start": {
+		"de": "Start automatic slide show",
+		"en": "Start automatic slide show",
+		"es": "Start automatic slide show",
+		"fr": "Start automatic slide show",
+		"ja": "Start automatic slide show",
+		"ko": "Start automatic slide show",
```

Comment:
looks like this wasn't translated?

Reply:
oh, this is deliberate, right?

Reply:
Correct, this is now generating the english for missing translations